mm2src/coins/eth.rs

```diff
@@ -2610,6 +2610,7 @@ impl MarketCoinOps for EthCoin {
             },
         };
 
+        let mut tx_hash: Option<H256> = None;
```

Comment:
There are duplications with the code here and the one in `find_taker_payment_spend_tx_impl` that I wanted refactored. Maybe we can find a way to unify them.

Reply:
Should be better
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2261/commits/e3b413225bfd87566987355992c4aa4f9eaeb4fc
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2261/commits/9007f123183ef60ef71ddc6789cb42ae49f04caa